styles.css

```diff
@@ -1,3 +1,861 @@
+  border-radius: 0.25rem;
+  transform: scale(1);
+  background-color: var(--secondary);
+  margin-bottom: 32px;
+}
+
+.keynote-speakers {
+  color: var(--primary);
+  padding: 5%;
+}
+
+.speakers-detail {
+  display: flex;
+  flex-direction: column;
+  gap: 2.5rem;
+}
+
+.speakers-heading {
+  text-align: center;
+}
+
+.speaker-persons {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+}
+
+.speaker-person {
+  display: flex;
+  gap: 3rem;
+}
+
+.speaker-image::before {
+  position: absolute;
+  top: -0.5rem;
+  inset-inline-start: -0.5rem;
+  width: calc(6 * 8px + 4px);
+  height: calc(6 * 8px + 4px);
+  content: '';
+  z-index: -34;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='8' height='8' viewBox='0 0 8 8'%3E%3Cg fill='%23000000' fill-opacity='.2'%3E%3Cpath fill-rule='evenodd' d='M0 0h4v4H0V0zm4 4h4v4H4V4z'/%3E%3C/g%3E%3C/svg%3E");
+}
+
+.bury {
+  display: none;
+}
+
+.speaker-image {
+  width: 7rem;
+  height: 7rem;
+  z-index: 64;
+  position: relative;
+}
+
+.speaker-info {
+  display: flex;
+  flex-direction: column;
+  gap: 0.25rem;
+}
+
+.speaker-name {
+  font-size: 1.3rem;
+  font-weight: 500;
+}
+
+.speaker-title {
+  color: var(--secondary-dark);
+  font-weight: 700;
+  padding-bottom: 0.75rem;
+  position: relative;
+}
+
+.speaker-biography {
+  color: var(--primary);
+  opacity: 1;
+  font-size: 0.9rem;
+}
+
+#more {
+  border: 1px solid var(--neutral);
+  border-radius: 0.25rem;
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  gap: 0.5rem;
+}
+
+.keynote-speakers #more i {
+  height: 1rem;
+  color: var(--secondary);
+}
+
+/* Partners (sponsors) */
+.sponsors {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  background-color: #333;
+  color: var(--neutral);
+  padding: 2rem 1rem;
+  margin-top: 32px;
+}
+
+.sponsers-heading {
+  color: var(--neutral);
+  text-align: center;
+}
+
+.sponser-underline {
+  width: 14%;
+  height: 2px;
+  background-color: var(--secondary);
+  margin: auto;
+}
+
+.sponsors-organization {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  gap: 2rem;
+}
+
+.sponsors img {
+  opacity: 0.5;
+  transition: opacity 300ms;
+}
+
+.sponsors img:hover {
+  opacity: 1;
+}
+
+.about-program:hover img {
+  transform: scale(1);
+  filter: blur(0.15rem);
+}
+
+/* Footer */
+.footer {
+  padding: 3rem 2rem;
+}
+
+.footer-main {
+  display: flex;
+  align-items: center;
+  gap: 1.2rem;
+}
+
+.footer-description {
+  color: var(--primary);
+  font-family: var(--sans-font);
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1em;
+  flex-shrink: 0;
+  flex-basis: 5rem;
+}
+
+.footer .footer-image img {
+  height: 130px;
+  width: 130px;
+  margin-left: -40px;
+}
+
+@media (min-width: 768px) {
+  /* .no-scroll {
+    overflow-y: auto;
+    overflow-x: hidden;
+  } */
+
```

Comment:
- [ ] Kindly make sure you remove all the commented code lines to make your code base cleaner.